container/container.go

```diff
@@ -2260,3 +2283,30 @@ func isAnaxNetwork(net *docker.Network, bridgeName string) bool {
 	}
 	return false
 }
+
+// Verify that the permission bits for the host side of the binding allow anyone/other
+// to access that file or directory.
+func hasValidBindPermissions(binds []string) error {
+	for _, bind := range binds {
+		containerVol := strings.Split(bind, ":")
+		if len(containerVol) > 1 && containerVol[1] != "" {
+
+			info, err := os.Stat(containerVol[0])
+			if err != nil {
+				return err
```

Comment:
There are a few cases where is should not return an error:
1. The item is a volume name which docker will create later.
2. The item is a directory name but the directory does not exist, docker will create it.
Here it should changed to `continue` instead of `return err`.

Reply:
Is there a way to discern those cases from a real error?